codewars/Tic-Tac-Toe Checker/TicTacToeChecker.txt

```diff
@@ -0,0 +1,29 @@
+def is_solved(board):
+    grid = [
+        [a1, a2, a3],
+        [b1, b2, b3],
+        [c1, c2, c3]
+    ] = board
+    
+    if a1 == a2 == a3 == 1: return 1

```

Comment:
actually it could be done little easier - by using idea that only some combinations leads to win